index.html

```diff
@@ -508,6 +508,24 @@ <h2 class="purple">Grapevine is...</h2>
 
         <section data-background="marvin-gaye.jpg">
           <h2>Thanks</h2>
+          <table>
+            <tr>
+              <td><img src="rt.jpg" class="headshot">
+              <td> Dr. Toal
+            <tr>
+              <td> <img src="grapevine-images/masao-pic.jpeg" class="headshot" width="100">
+              <td> Masao Kitamura
+            <tr>
+              <td><img src="grapevine-images/austin-pic.png" class="headshot" width="100">
+              <td> Austin Hentrup
+            <tr>
+              <td><img src="grapevine-images/austin-pic.png" class="headshot" width="100" style="opacity:0;">
+              <td> All the speakers
+          </table>
+        </section>
+
+        <section data-background="marvin-gaye.jpg">
+          <h2>Check us out at:</h2>
           <p style="margin-top:2em">https://github.com/lmucs/grapevine</p>
```

Comment:
I think the words "Check us out at" are superfluous. I think the final page should just have the url and "questions or thanks" or whatever should be implied.